vite.config.js

```diff
@@ -16,6 +16,7 @@ export default defineConfig({
       solid: {
         moduleName: "@lightningtv/solid",
         generate: "universal",
+        builtIns: [],
```

Comment:
Turns out without this, solid plugin will import Suspense from @lightningtv/solid and use it instead of existing function in the module.
Pretty much breaking language semantics.